recipes/trajan/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "trajan" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: a7b44ccd9898f45ac04033e46158f01748962ba9a943329ba0c0e27b59743296
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  entry_points:
+    - trajanshow = trajan.scripts.trajanshow:main
```

Comment:
Please add a test for the entry point:
https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#test-commands

Reply:
Done.